src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DataflowOption.cs

```diff
@@ -20,7 +20,7 @@ public static DataflowLinkOptions PropagateCompletion
         {
             get
             {
-                return new DataflowLinkOptions()
+                return new()
```

Comment:
may be easier to read if we keep this

Reply:
The type name is given just above. Perhaps if we used `get =>` here. I'll revisit this in the rewrite PR as it also changes this class, and it'll be easier to avoid conflicts.